data/p10/pdbg_compatible_propMapping.lsv

```diff
@@ -24,6 +24,9 @@ unit-mem_port:ibm,power10-memport
 lcard-dimm-ddimm:ibm,power10-dimm
 lcard-dimm-ddimm4u:ibm,power10-dimm
 lcard-dimm-ddr4:ibm,power10-dimm
+unit-ddr4-jedec:ibm,power10-dimm
+unit-ddr5-jedec:ibm,power10-dimm
+unit-ddr-jedec:ibm,power10-dimm
```

Comment:
I think it should be `unit-ddr` instead of `unit-ddr-jedec`

Reply:
Done